### ISSUE TITLE
explicit_H for charged ligands and protein

### ISSUE THREAD

**Issue by Le-Phung-Hien** [sc8668/GenScore#10]
Hi,

I just have some queston when running GenScore:

- I noticed there is an option for explicit_H in the genscore.py file. Just wonder if setting explicit_H=True will change the program, since a lot of my ligands are charged molecules. 
- Will --atom_contribution and --res_contribution make the final score more reliable?
- There are three files for GT_ft_0.5 in trained_models folder, which one should I use?

Thanks!

**Comment by sc8668** [comment]
1. Please set explicit_H=False since the models are trained with this setting,
2. No. These two options just provide some extra information for your complexes, i.e., decomposing the scores at either the atom or the residue level while their total score remains unchanged.
3. You can simple assess each model on your own system. Otherwise, perhaps the average.

**Comment by ackbar03** [comment]
Hi, 

Just to clarify, the hydrogen atoms still need to be specified in the SDF file for the ligand and the PDF file for the receptor right? I see in the example, the receptor and decoy files both have hydrogen atoms specified. 

Thanks

**Comment by sc8668** [comment]
No. Even if the hydrogens are specified in the SDF/PDB file, they will still be omitted when using the model.
